### DISCUSSION TITLE
How can I pass data to ContentFields?

### DISCUSSION THREAD

**Discussion by PiemP** [OrchardCMS/OrchardCore#14035] [Q&A]
Good evening, 

I would like to pass dynamic data to some ContentFields to precompile them. There is a way to do this?

Thank you

**Comment by ns8482e** [comment]
Not sure If I understand the question correctly - However there are some content events that's called when you create new content or load content from database.  You can create custom content handler that listen to such events and add dynamic data from different source to your content field/content item
